cvat/apps/annotation/ddln_spotter_importer.py

```diff
@@ -19,6 +25,96 @@ def iterate_frames(self):
             yield FrameReader(self._archive.open(path), frame_name, sequence_name)
 
 
+class CsvDirectoryImporter:
+    def __init__(self, directory_path):
+        self._path = Path(directory_path)
+
+    def iterate_frames(self):
+        for path in self._path.glob('*/*_y.csv'):
+            short_path = str(path.relative_to(self._path))
+            match = _filename_regex.match(short_path)
+            if not match:
+                continue
+            sequence_name, frame_name = match.groups()
+            yield FrameReader(path.open('rb'), frame_name, sequence_name)
```

Comment:
nit: why read in binary mode?

Reply:
Because python standard module `zipfile` allows to open files in archive only in binary mode (`zipfile` is used in  `CsvZipImporter`).

Having the file opened in binary mode both in `CsvZipImporter` and in `CsvDirectoryImporter` unifies the process, so I can use the same class - `FrameReader` in both cases and don't need to write another one.